articles/postgresql/howto-deny-public-network-access.md

```diff
@@ -12,6 +12,8 @@ ms.date: 03/10/2020
 
 This article describes how you can configure an Azure Database for PostgreSQL Single server to deny all public configurations and allow only connections through private endpoints to further enhance the network security.
 
+Note that clients from where connections to PostgreSQL Single server were made prior to the configuration described here will continue to have access to the server.
```

Comment:
```suggestion
When "Deny public access" is enabled, the new connection from the firewall rules will be rejected and existing clients can also not connect. If there is an open and active connection when this configuration is being changes , only that will be open since we cannot close that active open connection.  
```

Reply:
The statement previously is not true. Please use the suggestion to replace your line